.github/workflows/report.yml

```diff
@@ -42,6 +42,17 @@ jobs:
       - name: Generate Report
         run: poetry run -- nox -s project:report -- --format json | tee metrics.json
 
+      - name: Prepare files for sonar
+        run: |
+          poetry run -- coverage xml -o ci-coverage.xml --include="exasol/toolbox/*"
```

Comment:
I assume `--include="exasol/toolbox/*"` is specific for PTB.
How can other projects customize this?

Reply:
good catch, I think, this probably needs to be implemented via nox to make it generalizable